### DISCUSSION TITLE
PR scheduled but not created

### DISCUSSION THREAD

**Discussion by LukaszGrabarczyk-TomTom** [renovatebot/renovate#32296] [Request Help] [answered]
### How are you running Renovate?

A Mend.io-hosted app

### Please tell us more about your question or problem

I extended `renovate.json` like below and PR are not created automatically. In `Dependency Dashboard` there is a PR: "chore(deps): update all minor and patch updates"  with listed dependencies to be updated but this PR is under sth like `Awaiting Schedule`. I had to click checkbox to get this PR created.

Why it's not started/created as configured (at 2am) ?
Will it be always treated as immortal ?
Why this PR was labeled: 'Automerge: Disabled by config' ? (as config shows it is configured for grouped PR)
<img width="629" alt="image" src="https://github.com/user-attachments/assets/5f79778c-b748-4e11-8f9b-3de331ff1e33">

```
{
  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
  "extends": [
    "config:recommended"
  ],
  "packageRules": [
    {
      "matchUpdateTypes": ["patch", "minor"],
      "groupName": "All Minor and Patch Updates",
      "group": {"enabled": true},
      "prConcurrentLimit": 5,
      "automerge": true,
      "schedule": ["at 2am every weekday"]
    },
    {
      "matchPackageNames": ["*"],
      "separateMultipleMajor": true,
      "automerge": false
    }
  ]
}
```

### Logs (if relevant)

<details><summary>Logs</summary>

```

Replace this text with your logs, between the starting and ending triple backticks

```

</details>

**Comment by RahulGautamSingh** [comment, marked as answer]
1. Pkg rules are applied in order so the config of last matched rule overrides the ones before it. Hence, automerge false for all the deps ... due to your last rule
2. Config support for latex syntax in schedule is not supported (mentioned in docs) please try cron syntax and report back.
3. Regarding immortal prs please read the docs, its explained well in there.  [^1]

[^1]: https://docs.renovatebot.com/key-concepts/pull-requests/#immortal-prs

**Reply by rarkins** [reply]
Additionally read https://docs.renovatebot.com/key-concepts/scheduling/

**Reply by LukaszGrabarczyk-TomTom** [reply]
I reduced my config to:
```
{
  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
  "extends": [
    "config:recommended"
  ],
  "ignoreDeps": [
    "jakarta.persistence:jakarta.persistence-api",
    "com.fasterxml.jackson.core:jackson-annotations",
    "com.azure:azure-storage-queue"
  ],
  "packageRules": [
    {
      "matchUpdateTypes": ["patch", "minor"],
      "groupName": "All Minor and Patch Updates",
      "group": {"enabled": true},
      "automerge": true
    }
  ],
  "schedule": ["* 2 * * *"]
}
```

automerge indeed is now enabled but schedule still not works - prepared PR is in dashboard as Awaiting schedule but it's not created, what's wrong?

<img width="837" alt="image" src="https://github.com/user-attachments/assets/738443fb-adbe-4a95-acb3-3c51f233462b">

**Reply by rarkins** [reply]
You made your schedule too narrow. https://docs.renovatebot.com/key-concepts/scheduling/#in-repository-schedule-configuration

**Reply by LukaszGrabarczyk-TomTom** [reply]
ok, it's better then to create a 'window'  ?
`  "schedule": ["* 0-4 * * *"]`
1. Is it a must to also configure a timezone if I want scheduling working ?
2. I noticed that renovate makes scans often and earlier then this configured 2am (looking at dashboard and prepared PR). I thought it's triggered only at that configured time, so I'm confused how this works.

**Reply by rarkins** [reply]
Yes, a window.

1. Not needed
2. The Mend hosted app usually runs jobs on each repo every 3-4 hours, unless triggered sooner by events such as git commits or PR merges

**Reply by LukaszGrabarczyk-TomTom** [reply]
it works now, thank for help a lot
